docs/sitelist.md

```diff
@@ -723,6 +723,7 @@ The above is a reference list. Please see [the manualsearch doc](./manualsearch.
   - X Angels
   - Teen Sex Movs
 + #### Tonights Girlfriend | Matching type: *[Limited](./manualsearch.md#limited-search)* - **Actor only** 
++ #### ToughLoveX | Matching type: *[Limited](./manualsearch.md#limited-search)* - **Actor only** 
```

Comment:
You can actually update this to `Enhanced - Actor Only` as you're utilizing Date Match!